spec/rack_attack_dalli_proxy_spec.rb

```diff
@@ -2,9 +2,9 @@
 
 require_relative 'spec_helper'
 
-describe Rack::Attack::StoreProxy::DalliProxy do
+describe Rack::Attack::StoreProxies::DalliProxy do
   it 'should stub Dalli::Client#with on older clients' do
-    proxy = Rack::Attack::StoreProxy::DalliProxy.new(Class.new)
+    proxy = Rack::Attack::StoreProxies::DalliProxy.new(Class.new)
```

Comment:
I know this is not "officially" part of the public API, but given it might be easy to avoid in this case, can we please try not breaking any user code that references the current proxies, by keeping constant refs the same as before?

Maybe the `StoreProxy` class you added can become `BaseProxy` or just `Base`?

Reply:
Yes, sure this can be make. But if we will consider original part of this extracted pr, 1) this namespace will still be changed to something like `StoreAdapters` and 2) personally, new names (`StoreProxies` namespace and `StoreProxy` base class) looks cleaner to me than `Base` or `BaseProxy`.
 
So, taking into account first mentioned reason, should I still preserve original namespace name?